modules/order/order.graphql.js

```diff
@@ -89,8 +94,8 @@ input OrderUserInput {
 }
 
 input CurrencyInputSet {
-  currency: String!
-  value: Float!
+  currency: String
+  value: Float
 }
 
 input DeliveryInput {
```

Comment:
You need to have `enum` on `sendBy` field on this input

Reply:
fixed